capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
![image](https://github.com/ngeelani48/oop-school-library/assets/93039359/071b5983-3107-4f43-bfea-fbb8828733e1)

- [ ] It would be nice if you fix this error shown in the image above to make your app work as expected :+1: I have faced this error while running the below queries :arrow_down: Kindly run them and check if you are facing the same error :thinking:

```ruby
 person = Person.new(22, 'maximilianus')
 person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 capitalizedTrimmedPerson.correct_name
```